samples/event-sourcing-esdb-simple/src/main/java/io/eventdriven/ecommerce/shoppingcarts/addingproductitem/AddProductItemToShoppingCart.java

```diff
@@ -0,0 +1,45 @@
+package io.eventdriven.ecommerce.shoppingcarts.addingproductitem;
+
+import io.eventdriven.ecommerce.pricing.ProductPriceCalculator;
+import io.eventdriven.ecommerce.shoppingcarts.Events;
+import io.eventdriven.ecommerce.shoppingcarts.ShoppingCart;
+import io.eventdriven.ecommerce.shoppingcarts.productitems.ProductItem;
+
+import java.util.UUID;
+
+public record AddProductItemToShoppingCart(
+  UUID shoppingCartId,
+  ProductItem productItem,
+  Long expectedVersion
+) {
+  public static AddProductItemToShoppingCart From(UUID cartId, ProductItem productItem, Long expectedVersion) {
```

Comment:
```suggestion
  public static AddProductItemToShoppingCart from(UUID cartId, ProductItem productItem, Long expectedVersion) {
```
You're in the JVM world, don't do it :)...
Why? Because the static classes also starts with capital letter.

Reply:
I know, just fixed it: https://github.com/oskardudycz/EventSourcing.JVM/pull/1/commits/987a08f58e8584b2a1b06dfb9fba88d013716431 :)